### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by gemfield**
Add Docker support.

### CODE REVIEW

File: docker/ubuntu-16.04/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM  ubuntu:16.04
```
Comment (dpocock) [inline docker/ubuntu-16.04/Dockerfile]: I'm using Debian buster for things like release management, would you like to test Docker with a Debian buster image too?

File: docker/ubuntu-16.04/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        DEBIAN_FRONTEND="noninteractive" \
+        apt-get install -y vim git cmake g++ autoconf libtool gperf pkg-config binutils tzdata && \
+        apt-get autoremove -y && \
+        apt-get clean -y
+
+ENV RESIP_DIR=/opt/resiprocate
+WORKDIR /workspace
+RUN git clone https://github.com/resiprocate/resiprocate
+RUN cd resiprocate && \
+          git submodule update --init --recursive && \
```
Comment (dpocock) [inline docker/ubuntu-16.04/Dockerfile]: did you see the build/debian.sh script?

Comment (gemfield) [reply docker/ubuntu-16.04/Dockerfile]: Yes, I will update the Dockerfile according to this script.

Comment (gemfield) [reply docker/ubuntu-16.04/Dockerfile]: @dpocock is there a apt way to install libsrtp-dev on Debian buster(seems only libsrtp2 can be found on this release...)?
Now there has a relate compile issue:
```bash
Flow.hxx:16:10: fatal error: srtp/srtp.h: No such file or directory
```

**Review by kingabnt — COMMENTED** [review]
Duplicate of #

**Pull request closed without merging**